conf/modules.config

```diff
@@ -80,7 +78,11 @@ process {
     }
 
     withName: FUSIONINSPECTOR {
-        ext.when         = { !params.skip_vis }
+        ext.when = { !params.skip_vis }
+    }
+    withName: FUSIONINSPECTOR_DEV {
+        ext.when = { !params.skip_vis }
+        ext.args =  '--STAR_xtra_params "--limitSjdbInsertNsj ${params.fusioninspector_limitSjdbInsertNsj"'
```

Comment:
I see, so it's just working with the dev version of fusion inspector.
You still want to keep both version of the tools here?